kr8s/tests/test_objects.py

```diff
@@ -597,6 +597,28 @@ async def test_pod_port_forward_start_stop(nginx_service):
     assert pf._bg_task is None
 
 
+async def test_multiple_pods_port_forward_start_stop(nginx_service):
```

Comment:
This test effectively just runs the `test_pod_port_forward_start_stop` test twice. So I'm not sure this adds any benefit over the existing test.

If the goal is just to run the port forward multiple times in a row to reproduce the issue maybe it would be better to just put a `for _ in range(5):` in the existing test to run it a few times.